docpages/03_example_programs.md

```diff
@@ -594,10 +594,10 @@ To add a slash command you should use the dpp::cluster::global_command_create me
 or dpp::cluster::guild_command_create to create a local command (available only to one guild).
 
 When a user issues these commands the reply will arrive via the `on_slashcommand` event which you can hook, and take action
-when you see your commands. It is possible to reply to an interaction by using either the dpp::interaction_create_t::reply method,
+when you see your commands. It is possible to reply to an interaction by using either the dpp::slashcommand_t::reply method,
```

Comment:
technically the reply method is in the parent class interaction_create_t so this will properly ref link to the method docs. changing this might break it as the reply method is inherited into slashcommand_t

Reply:
alright, ill revert it